docs/advanced_examples/LoraMLP.ipynb

```diff
@@ -41,10 +40,8 @@
     "from sklearn.datasets import make_circles, make_moons\n",
     "from torch import nn, optim\n",
     "from torch.utils.data import DataLoader, TensorDataset\n",
-    "from tqdm import tqdm\n",
     "\n",
-    "from concrete.ml.torch.hybrid_model import HybridFHEModel\n",
-    "from concrete.ml.torch.lora import LoraTraining, get_remote_names\n",
+    "from concrete.ml.torch.lora import LoraTrainer\n",
     "\n",
     "# Set random seed for reproducibility\n",
     "SEED = 42\n",
```

Comment:
Two synthetic datasets are generated: one for the original training and a second one for LoRA fine-tuning.

Reply:
in # Task 1: Two interleaving half circles (make_moons)

maybe remove the (make_moons) ?

Reply:
removed